src/dbnode/storage/shard.go

```diff
@@ -816,10 +816,16 @@ func (s *dbShard) writeAndIndex(
 		commitLogSeriesID          ident.ID
 		commitLogSeriesTags        ident.Tags
 		commitLogSeriesUniqueIndex uint64
+		shouldWrite                = true
```

Comment:
`shouldWrite` -> `wasWritten`

Also, seems more intuitive for default value to be `false`. Does that break something?

Reply:
Oh I see why you default it to `true`. Want to add a comment about the default value I.E we err on the side of caution and want to write it to the commitlog always in the async case because we don't know if its gonna be written or not until later?

Reply:
Sure sounds good, added comment